scripts/quests/bastok/Return_to_the_Depths.lua

```diff
@@ -0,0 +1,256 @@
+-----------------------------------
+-- Return to the Depths
+-- Ayame
+-- Log ID = 1
+-- Quest ID = 78
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/quests')
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require("scripts/globals/status")
+require('scripts/globals/titles')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.BASTOK, xi.quest.id.bastok.RETURN_TO_THE_DEPTHS)
+
+quest.reward =
+{
+    fame     = 10,
```

Comment:
Could probably increase the fame received. 10 is similar to those repeatable quests like zeruhn soot

Reply:
Updated to 60.  An estimate either way, and you also get the bonus of the ahriman tears warp now.